### PR TITLE
fix: set correct link to setup-gcloud

### PR DESCRIPTION
**Pull request by danBamikiya**

### CODE REVIEW

**Comment by danBamikiya** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -169,4 +171,4 @@ Credentials.
 [sa]: https://cloud.google.com/iam/docs/creating-managing-service-accounts
 [gh-runners]: https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners
 [gh-secret]: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
-[setup-gcloud]: ../setup-gcloud
+[setup-gcloud]: ../../../setup-gcloud
```
Comment (averikitsch) [inline README.md:174]: Do you mind updating this to:
```suggestion
[setup-gcloud]: https://github.com/google-github-actions/setup-gcloud
```

**Review by averikitsch — APPROVED** [review]
Thank you!

File: README.md (hunk 1 above)
Comment (danBamikiya) [reply README.md]: > Thank you!

You're welcome!

**Pull request merged**